Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/apple/swift-log.git",
         "state": {
           "branch": null,
-          "revision": "57c6bd04256ba47590ee2285e208f731210c5c10",
-          "version": "1.3.0"
+          "revision": "173f567a2dfec11d74588eea82cecea555bdc0bc",
+          "version": "1.4.0"
```

Comment:
we normally do not commit this file

Reply:
Okey will delete 👍

Reply:
https://github.com/slashmo/gsoc-swift-baggage-context/pull/36